test-utils/src/main/resources/detect-routable-ip.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Script which returns your primary IP; works on Mac OS X and Linux
+#
+
+case $(uname) in
```

Comment:
@timcharper what is the reason for this being as script and not Scala code?

Reply:
The motivation at the time I wrote this was I could write this script and just run it on different distros in docker and check if it worked :)

If we can write equivalent Scala code then no problem there.

Reply:
Hm, `java.net.InetAddress.getLocalHost.getHostAddress()` returns the same as `detec-routable-ip.sh`. At least on my machine.